Cargo.toml

```diff
@@ -111,3 +111,9 @@ wasm-bindgen-crate = { version = "0.2.58", package = "wasm-bindgen" }
 # tokio = { path = "../tokio/tokio" }
 # mio = { git = "https://github.com/tokio-rs/mio.git" }
 # h2 = { git = "https://github.com/hyperium/h2.git" }
+
+[profile.release]
+lto = true
+codegen-units = 1
+opt-level = "s"
```

Comment:
Do we want `s` or `3` here? I'm a little uncomfortable with optimizing for size, when I think we in most cases want to optimize for speed.